spec/Body/Body.vspec

```diff
@@ -161,11 +161,53 @@ Windshield.WasherFluid.Level:
 ##
 Lights:
   type: branch
-  description: All lights.
+  instances: ["Beam", "Running", "Backup", "Parking", "Brake", "Fog", "Hazard", "Indicator"]
+  description: Exterior lights.
 
-# Include lights specification and attach it to the Lights branch
-#include ExteriorLights.vspec Lights
+Lights.Beam:
+  type: branch
+  instances: ["Low","High"]
+  description: Beam lights
+# Include lights specification and attach it to the Lights.Beam branch
+#include BeamLights.vspec Lights.Beam
+
+Lights.Running:
+  type: branch
+  description: Running lights
+# Include lights specification and attach it to the Lights.Running branch
+#include RunningLights.vspec Lights.Running
+
+Lights.Backup:
+  type: branch
+  description: Backup lights
+# Include lights specification and attach it to the Lights.Backup branch
+#include BackupLights.vspec Lights.Backup
+
+Lights.Parking:
+  type: branch
+  description: Parking lights
+# Include lights specification and attach it to the Lights.Parking branch
+#include ParkingLights.vspec Lights.Parking
+
+Lights.Fog:
+  type: branch
+  instances: ["Rear","Front"]
+  description: Fog lights
+# Include lights specification and attach it to the Lights.Fog branch
+#include FogLights.vspec Lights.Fog
 
+Lights.Hazard:
+  type: branch
+  description: Hazard lights
+# Include lights specification and attach it to the Lights.Hazard branch
+#include HazardLights.vspec Lights.Hazard
```

Comment:
Hazard lights blink. Need to be clear what is meant by hazard lights being on. That is, do we mean "activated" or do we expect to receive a cycling signal telling us on/off/on/off/...